### DISCUSSION TITLE
Error 413 with Budibase File Upload plugin

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#9630] [Help] [answered]
Hello,

Not sure if this is the correct place for this.

I am having an issue with the File Upload plugin. When I try to upload something about 6mb in size I get HTTP error 413, I have changed the client_max_body_size in nginx to 0 on both the reverse proxy and bbproxy container. I have also set max_allowed_packet=1073741824 in MariaDB.

I enabled the logs on the bbproxy container and the only thing that appears is `2023/02/09 15:11:41 [warn] 26#26: *43 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000001, client: 172.18.0.1, server: _, request: "POST /api/datasource_plus_6009a27a119e48d8878bfd2a5b6eec86__TableName/rows HTTP/1.0", host: "server.company.com", referrer: "https://server.company.com/app/appname"`

Is there a setting that I need to change in bbapps or another container to enable larger upload sizes.

Thank you.

**Comment by melohagan** [comment, marked as answer]
Hey @FlaminWrap 

Nice job figuring out what the issue was: https://github.com/chungchunwang/Budibase-File-Upload/issues/2#issuecomment-1427873840

As Michael noted, he has a PR out which should allow you to define these limits through an environment variable!
